client/plugins/postcss-image-path.js

```diff
@@ -0,0 +1,38 @@
+const postcss = require('postcss');
+const path = require('path');
+
+// opts example: {helper : 'image-path', assetPath: '/app/assets/images/'})
+// TODO expand helper to check all Rails asset helpers.
+module.exports = postcss.plugin('postcss-image-path', function (opts) {
+  opts = opts || {};
+  const imagePathHelper = opts.helper || 'image-path';
+  const assetPath = opts.assetPath || process.cwd();
+
+  return function (css, result) {
+    css.walkDecls(function (decl) {
+      // Return if css declaration value part doesn't contain Rails helper
+      // e.g. image-path('image.png')
+      if (!decl.value || decl.value.indexOf( imagePathHelper + '(') === -1) {
```

Comment:
Can the `decl.value` se something else than a String? If it is truthy, but not a String, the `.indexOf` method might not be found.